src/humancellatlas/data/metadata/helpers/json.py

```diff
@@ -3,7 +3,7 @@
 
 from dataclasses import field, fields, is_dataclass
 
-from humancellatlas.data.metadata import Entity
```

Comment:
I guess the api change requires us changing the import on Green Box at the same time right? https://github.com/HumanCellAtlas/pipeline-tools/blob/0f04daf7850167fab2b8968ddafaaeaf48d8bd7d/pipeline_tools/input_utils.py#L4

Reply:
Oh, yes, I meant to mention this to you. It was a mistake to put everything into `…/metadata/__init__.py`